pandas/conftest.py

```diff
@@ -376,28 +376,40 @@ def unique_nulls_fixture(request):
              FixedOffset(0), FixedOffset(-300), timezone.utc,
              timezone(timedelta(hours=1)),
              timezone(timedelta(hours=-1), name='foo')]
+TIMEZONE_IDS = ['None', 'UTC', 'US/Eastern', 'Asia/Tokyp',
+                'dateutil/US/Pacific', 'dateutil/Asia/Singapore',
+                'dateutil.tz.tzutz()', 'dateutil.tz.tzlocal()',
+                'pytz.FixedOffset(300)', 'pytz.FixedOffset(0)',
+                'pytz.FixedOffset(-300)', 'datetime.timezone.utc',
+                'datetime.timezone.+1', 'datetime.timezone.-1.named']
```

Comment:
Remind me again why `TIMEZONE_IDS` are needed?

Reply:
@gfyoung 
Fair point (as with the other conftest-related comments), but it was needed for me because debugging tz-tests is very tedious without knowing for which timezone the tests failed. 

Related discussion: https://github.com/pandas-dev/pandas/pull/25425#discussion_r263968845

Reply:
Got it.  Can you open a PR that breaks off the following changes:

* `TIMEZONE_IDS`
* `DATETIME_DTYPES` rename
* `TIMEDELTA_DTYPES` rename

It will be a small PR, but I think it's a reasonable change to be made independently of this PR and shrink it by a little bit.